controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -576,7 +576,8 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           };
         }
 
-        const newSchemaSDL = new TextDecoder().decode(req.schema);
+        // `{ deleteSubgraph }` allows us to create a diff on a 'deleted', empty schema.
+        const newSchemaSDL = req.checkDeletion ? '{ deleteSubgraph }' : new TextDecoder().decode(req.schema);
```

Comment:
Creating a dummy schema should not be necessary when the subgraph is not part of the composition and diffing.

Reply:
I don't see how, it actually diffs between the sdl of that specific graph
```
const schemaChanges = await getDiffBetweenGraphs(subgraph.schemaSDL, newSchemaSDL);
```

Reply:
updated getDiffBetweenGraphs to allow to check against a 100% empty / deleted schema.